Project/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -95,7 +95,7 @@ public override void Initialize()
     /// <param name="pos"></param>
     void SpawnTarget(Transform prefab, Vector3 pos)
     {
-        m_Target = Instantiate(prefab, pos, Quaternion.identity, transform);
+        m_Target = Instantiate(prefab, pos, Quaternion.identity);
```

Comment:
```suggestion
        m_Target = Instantiate(prefab, pos, Quaternion.identity, transform.parent);
```
I think spawning at the root will make the scene hierarchy messy. This will spawn under the area gameobject

Reply:
Updated.